packages/contentstack-import-setup/README.md

```diff
@@ -47,7 +47,7 @@ $ npm install -g @contentstack/cli-cm-import-setup
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-cm-import-setup/1.3.0 darwin-x64 node-v22.17.0
+@contentstack/cli-cm-import-setup/1.3.0 darwin-arm64 node-v22.14.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.